core/server/lib/image/manipulator.js

```diff
@@ -8,60 +8,20 @@ const fs = require('fs-extra');
  * We currently can't enable compression or having more config options, because of
  * https://github.com/lovell/sharp/issues/1360.
  */
-const process = (options = {}) => {
-    let sharp, img, originalData, originalSize;
-
-    try {
-        sharp = require('sharp');
-    } catch (err) {
-        return Promise.reject(new common.errors.InternalServerError({
-            message: 'Sharp wasn\'t installed',
-            code: 'SHARP_INSTALLATION',
-            err: err
-        }));
-    }
-
-    // @NOTE: workaround for Windows as libvips keeps a reference to the input file
-    //        which makes it impossible to fs.unlink() it on cleanup stage
-    sharp.cache(false);
 
+const unsafeProcess = (options = {}) => {
```

Comment:
:heart: how tiny the method has become!

Reply:
the ` withoutEnlargement: true` reduced it by one `if` and comparing two buffers in one method saved another bit. nice!

Reply:
:grinning: glad you like!!!

Reply:
I LIKE TOO 💥